### ISSUE TITLE
Hosting Jupyter Studio days in April [SF] and August [NY]

### ISSUE THREAD

**Issue by dharmaquark** [jupyterlab/team-compass#183]
### Problem

Bloomberg would like to host a few studio days in 2023! I'd like to get a sense of which months/weeks would make sense for the wider community and whether weekdays or weekends would be more accessible to attendees. I'd also be interested in working with the community to advertise the events ahead of time through different channels to make these as inclusive and open as possible to anyone interested in joining!

### Proposed Solution

1. San Francisco-hosted event in April
- would likely host this in Bloomberg's downtown 140 New Montgomery Office
- Potential Dates: week (or weekend) of 4/17, 4/24
  - would early or late in the week be more ideal? or a Saturday/Sunday date?

2. New York-hosted event in April
- would likely host this in one of Bloomberg's New York offices (where we can book a larger meeting space for the full day)
- Potential Dates: week of 8/21, 8/28
  - it may be more difficult in New York to coordinate a weekend event

### Additional context

I will be joining the next few Jupyter dev meetings to get additional feedback ahead of confirming dates and starting to advertise any studio days more officially/widely! Thank you in advance for any advice/suggestions on next steps 😃

**Comment by jasongrout** [comment]
FYI, [PyCon US](https://us.pycon.org/2023/) is April 19-27, 2023. Not sure if you want to work around that, but at least you should be aware.

**Comment by dharmaquark** [comment]
Proposing moving SF studio from April to June so that we can also announce/advertise during May's JupyterCon

**Comment by dharmaquark** [comment]
added post in discourse for the August Studio Day: https://discourse.jupyter.org/t/jupyter-studio-day-in-nyc-august-28th/20929 , date for San Francisco event TBD - may be hosting that either later in the year or early next year, depending on timing (want to avoid scheduling during winter holidays!)

**Comment by andrii-i** [comment]
Hi @dharmaquark. I'd love to participate in SF event and maybe even give a talk and/or host a workshop if you'd be open to that.

**Comment by dharmaquark** [comment]
> Hi @dharmaquark. I'd love to participate in SF event and maybe even give a talk and/or host a workshop if you'd be open to that.

yeah, we'd definitely like to plan for the SF event to be as open as possible, and have space for informal demos/presentations during the day- I can keep this thread updated as we finalize the date for the SF Studio day!
